core/tee/tee_svc.c

```diff
@@ -121,6 +121,14 @@ TEE_Result tee_svc_sys_get_property(uint32_t prop, tee_uaddr_t buf, size_t blen)
 	 */
 	static const uint32_t sys_time_prot_lvl = 100;
 	static const uint32_t ta_time_prot_lvl = 100;
+	static const bool crypto_ecc_en = false;
+	static const uint32_t ts_antiroll_prot_lvl = 0;
+	static const char trustedos_impl_version[] = "Version 0.0";
+	static const uint32_t trustedos_impl_bin_version = 0x00000000;
+	static const char trustedos_manufacturer[] = "Linaro";
+	static const char fw_impl_version[] = "Version 0.0";
+	static const uint32_t fw_impl_bin_version = 0x00000000;
+	static const char fw_manufacturer[] = "Linaro";
```

Comment:
Version is not terribly useful as is. IMO we should extract the info dynamically from Git. Check `git describe --dirty` for instance. Or at least, define configuration variables in `mk/config.mk`.
Also the symbols should probably be `__weak` so that they may be overridden by platform code.
Finally, some of them had better not be static because they may be useful outside this file (for instance `trustedos_impl_version` may be used to log a message on startup).